pyproject.toml

```diff
@@ -57,7 +57,7 @@ dependencies = [
     "torchvision>=0.14.1",
     "torchmetrics[image]>=1.0.1",
     "typing_extensions>=4.4.0",
-    "viser==0.1.27",
+    "viser==0.2.1",
```

Comment:
maybe we can actually bump to `0.2.2` once https://github.com/nerfstudio-project/viser/pull/223 is merged, since that resolves some unlikely but plausible race conditions in the messaging infra!

Reply:
bumped!